TurkishFilter/sections/css_extended.txt

```diff
@@ -5,6 +5,10 @@ vizyonfilmizle.tv#$#.fancybox-overlay-fixed[-ext-has="> div"] { display: none!im
 vizyonfilmizle.tv#$#.fancybox-lock body[-ext-has="> div"] { overflow: visible!important; }
 vizyonfilmizle.tv#$#html[-ext-has="> body"] { overflow: visible!important; }
 !
+itemci.com#?#body > a[target="_blank"]:has( > #bg_rek)
```

Comment:
#bg_rek is enough as the link itself is not over the full page like #bg_rek.

Reply:
https://github.com/AdguardTeam/AdguardFilters/commit/5e769da113cf1aeb35dcaf76cac524a7aed62220